### DISCUSSION TITLE
How to add an address to a contacts list

### DISCUSSION THREAD

**Discussion by ThomasFreedman** [markqvist/Reticulum#410] [Nomad Network]
It's been a few months since I've logged into the forum, but it appears as the old forum has been moved to here on github. I get a 404 "not found" error going to https://unsigned.io/forum/login.php

In February I revised my 2 reticulum relay nodes, adding a BBS and figuring things out a bit. I managed to save the lxmf address for the  BBS on each node to the other node (if on node A it has the address to node B, and node B has the address to node A). I don't recall how I did that and I couldn't find an applicable keyboard shortcut or reference to saving addresses.

I'm about to release a beta version of my new "CensorGuard" program and I'd like to preconfigure the included nomadnet client to include these 2 BBS lxmf addresses. I've got too much to do to stop & figure it out again, so I'm hoping someone here will jog my memory and explain how to save an address in nomadnet so it appears in the list of conversations or list of nodes. I believe the BBS is programmed to listen to a std peer address and post the msg on the node.

Happy Holidays to everyone!

**Comment by ThomasFreedman** [comment]
I figured it out... again. While on conversations tab use ^n to bring up a dialog to add a peer address.

**Reply by LinuxinaBit** [reply]
Yeah, the documentation could be a bit better.
I accidentally told somebody to use ^u :facepalm:

**Reply by markqvist** [reply]
Agreed. If anyone is up for writing a short snippet that fits into some place of the existing Guide section, it would be great to include that in the next release!
